lib/claim/fees-claimant.js

```diff
@@ -127,11 +142,17 @@ class FeesClaimant {
      * Get the withdrawable amount of fees
      * @param {Wallet} wallet - The claimer nahmii wallet
      * @param {Currency} currency - The currency
-     * @returns {Promise} A promise that resolves into a BigNumber value representing the withdrawable amount of fees.
+     * @returns {Promise} A promise that resolves into a string value representing the withdrawable amount of fees.
      */
     async withdrawableFees(wallet, currency) {
         const ensemble = _tokenHolderRevenueFundContractEnsemble.get(this);
-        return ensemble.stagedBalance(wallet, currency);
+        const provider = _provider.get(this);
+
+        const withdrawableFees = await ensemble.stagedBalance(wallet, currency);
+
+        const tokenInfo = await provider.getTokenInfo(currency.ct.toString(), true);
```

Comment:
These two lines of currency-formatting has been repeated three times in the code above.
Would that make it a candidate for factoring out into a utility function?

Reply:
👍